hack/install-database.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: databases
```

Comment:
The CI is more production like, so it could make sense that the DB uses a different namespace. We don't need to have the database anywhere special, as long as its accessible.

I noticed that this causes an issue with the `TestDatabaseConnection` test, I'm working on it.

Reply:
After some back and forth with a couple ideas, I just changed the namespace and name of the deployment being manipulated to the one on `databases` and added a PV and PVC to the database deployment, so the initialization (so the user and password created for KubeArchive) is retained.